string Command
float64 Velocity